README.md

```diff
@@ -0,0 +1,9 @@
+## Build
+
+```
+$ ./autogen.sh
```

Comment:
Is that a pre-requisite? I know some source distributions ship with the result of this, but equally the result is an autogenerated file in the source repo.